cmd/mount.go

```diff
@@ -400,7 +403,27 @@ var mountCmd = &cobra.Command{
 			}
 
 			ctx, _ := context.WithCancel(context.Background()) //nolint
-			daemon.SetSigHandler(sigusrHandler(pipeline, ctx), syscall.SIGUSR1, syscall.SIGUSR2)
+			var sigusr2, sigchild chan os.Signal
+			if !daemon.WasReborn() { // execute in parent only
+				// redirect libfuse stderr to a temp file
+				f, err := ioutil.TempFile("", "libfuse_stderr_")
+				if err != nil {
+					log.Err("failed to create temp file for storing libfuse error log [%v]", err)
+				} else {
+					defer os.Remove(f.Name())
+					dmnCtx.LogFileName = f.Name()
+				}
+
+				// set signal handler
+				sigusr2 = make(chan os.Signal, 1)
+				signal.Notify(sigusr2, syscall.SIGUSR2)
+				sigchild = make(chan os.Signal, 1)
+				signal.Notify(sigchild, syscall.SIGCHLD)
+			} else { // execute in child only
+				daemon.SetSigHandler(sigusrHandler(pipeline, ctx), syscall.SIGUSR1, syscall.SIGUSR2)
+				go daemon.ServeSignals()
```

Comment:
I've separated signal registration for child and parent process, also add `daemon.ServeSignals` for child process, otherwise signal registration won't take effect in child process.